community/README.md

```diff
@@ -68,6 +68,7 @@ Policy  | Description | Prerequisites
 [Example of configuring DNS with a policy](./CM-Configuration-Management/policy-cluster-dns-sample.yaml) | Use this policy to configure DNS in your OpenShift cluster. For example, you can remove public DNS. | OpenShift 4.x is required. See the [OpenShift Documentation](https://docs.openshift.com/container-platform/latest/post_installation_configuration/network-configuration.html#private-clusters-setting-dns-private_post-install-network-configuration) This policy is only valid for OpenShift 4.x and needs to be adjusted for the proper environment.
 [Example of configuring the Cluster Network Operator with a policy](./CM-Configuration-Management/policy-cluster-network-sample.yaml) | Use this policy to configure the network of your OpenShift cluster. | OpenShift 4.x is required. See the [OpenShift Documentation.](https://docs.openshift.com/container-platform/latest/post_installation_configuration/network-configuration.html#nw-operator-cr_post-install-network-configuration) This policy is only valid for OpenShift 4.x and needs to be adjusted for the proper environment.
 [Example of creating a deployment object](./CM-Configuration-Management/policy-nginx-deployment.yaml) | This example generates 3 replicas of \`nginx-pods\`. | See the [Kubernetes Documentation](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) to learn more about Deployments.
+[Example of creating a xcrypt deployment object](./CM-Configuration-Management/policy-ztc-cmc-deployment.yaml) | This example deploys a replica of \`zts-cmc-pod\`. | See the [Kubernetes Documentation](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) to learn more about Deployments.
```

Comment:
Instead of the README.ACM can there be a link to ZTS documentation here instead

Reply:
https://github.com/stolostron/policy-collection/pull/304/files

I created a new pull request that has only 2 files.  The PR number is 304.